src/web/core/src/main/java/org/geoserver/web/GeoServerHomePage.java

```diff
@@ -252,12 +252,8 @@ void selectHomePage(String workspaceName, String layerName) {
         String layerSelection = toLayer(workspaceName, layerName);
 
         PageParameters pageParams = new PageParameters();
-        if (workspaceSelection != null) {
-            pageParams.add("workspace", workspaceSelection, 0, INamedParameters.Type.QUERY_STRING);
-        }
-        if (layerSelection != null) {
-            pageParams.add("layer", layerSelection, 1, INamedParameters.Type.QUERY_STRING);
-        }
+        pageParams.add("workspace", workspaceSelection, 0, INamedParameters.Type.QUERY_STRING);
+        pageParams.add("layer", layerSelection, 1, INamedParameters.Type.QUERY_STRING);
```

Comment:
I still would love to use ``INamedParameters.Type.PATH`` but could not figure out the mapping.